.github/actions/e2e/action.yml

```diff
@@ -0,0 +1,32 @@
+---
+ name: Test E2E
+ description: Run e2e tests
+ inputs:
+   working_directory:
+     description: "Working directory"
+     required: false
+     default: .
+ runs:
+   using: "composite"
+   steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+       with:
+         repository: neptune-ai/neptune-lightgbm
+         path: ${{ inputs.working_directory }}
+
+     - name: Install dependencies
+       working-directory: ${{ inputs.working_directory }}
+       run: pip install -e .[dev] && pip install scikit-learn
```

Comment:
No need to change it now, but FYI, you can also format it as multiline for improved readability:

```
run: |
   pip install -e .[dev]
   pip install scikit-learn
```

Reply:
Thanks!